index.html

```diff
@@ -18,42 +18,277 @@
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+			<div class="sec-2 center">
+				<h3 class="h3-light">
+					Profesional Art <br>
+					Printing Data
+				</h3>
+				<p class="p-light">
+					A daily selection of privately <br>
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+			<div class="sec-2 center">
+				<h3 class="h3-light">
+					Profesional Art <br>
+					Printing Data
+				</h3>
+				<p class="p-light">
+					A daily selection of privately <br>
+					personalized reads; no accounts or <br>
+					sign-ups required. Has been the <br>
+					industry's standard.
+				</p>
+				<ul class="list-icons">
+					<li>
+						<a href="#"><img src="Works-Images/html2.png"></a>
+						<a href="#"><img src="Works-Images/bootstrap2.png"></a>
+						<a href="#"><img src="Works-Images/Ruby2.png"></a>
+					</li>
+				</ul>
+				<button class="btn" type="button">See Project</button>
+			</div>
+		</section>
+		<section class="Bottom">
+			<section class="Aboutme">
+				<h2 class="h2-orange">
+					About me
+				</h2>
+				<p class="p-light">
+					Hello I’m a software developer! I can help <br>
+					you build a product , feature or website <br>
+					Look through some of my work and <br>
+					experience! If you like what you see and <br>
+					have a project you need coded, don’t <br>
+					hesitate to contact me.
+				</p>
+				<button type="button" class="btn">Get my resume</button>
+			</section>
+			<section class="Techs">
+				<div class="languages center">
+					<hr>
+					<ul>
+						<li class="diamond large symbol ">Languages</li>
+						<ul class="no-style">
+							<li>JavaScript</li>
+							<li>Ruby</li>
+							<li>HTML</li>
+							<li>CSS</li>
+						</ul>
+					</ul>
+					<ul>
+						<li class="large symbol">Frameworks</li>
+						<ul class="no-style">
+							<li>React.js</li>
+							<li>Ruby on Rails</li>
+							<li>RSpec</li>
+							<li>Capybara</li>
+						</ul>
+					</ul>
+					<ul>
+						<li class="large circle">Skills</li>
+					</ul>
+					<ul class="skill">
+						<li>Database Management</li>
+						<li>Version Control</li>
+						<li>CLI</li>
+						<li>API Design</li>
+						<li>Web Development</li>
 					</ul>
-					
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md), please use `ul` and `li` for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.


Example
```HTML
<ul>
  <li>
    <h3>Languages</h3>
    . . . 
    <ul>
       <li>JavaScript</li>
       <li> . . .  </li>
       <li> . . .  </li>
    </ul>
  </li>
  <li>
    <h3>Frameworks</h3>
     . . . 
  </li>
  <li> 
   <h3>Skills</h3>
    . . . 
  </li>
</ul>
```

Reply:
@abdulhamiid Can you please explain a bit more specifically about using ul li in the Aboutme section because I am already using ul li and to me it looks very similar to the example code you have provided. Can you explain a bit more specific about where I'm wrong so I can fix it. Thanks